### ISSUE TITLE
Dev feature: DB Migrations and Fixture loading

### ISSUE THREAD

**Issue by osfameron** [mysociety/fixmystreet#856]
After discussing with @davea, it would be good to:
- [ ] use a migration toolkit such as https://metacpan.org/pod/DBIx::Class::Migration to make it easier to make DB changes
- [ ] use a Fixtures tool (cp "factory boy/girl") like https://metacpan.org/pod/DBIx::Class::Fixtures to be able to dump and reload fixtures for testing/dev/demos

**Comment by osfameron** [comment]
In parallel to the above (which is a bit baroque) I've been slowly fixing the bin/migrate (e.g. fixing upgrades, which were broken, and adding a downgrade feature.)  So I suspect the DBIC migrations is less urgent, however...

**Comment by osfameron** [comment]
... being able to set up realistic data for a cobrand would be very useful (e.g. as discussed with @zarino today re ZWN).

 - [ ] copy or create realistic recent data (e.g. fixtures.  if copied, these should be anonymised)
 - [ ] create bodies and realistic categories (there is some WIP on this in zurich-2015-dev branch)
 - [ ] create/copy a handful of realistic photos for sample reports (not Terry Wogan)
 - [ ] create from_body users for all relevant bodies.

**Comment by dracos** [comment]
The migrations seem to work okay. There is now a fixture script for the UK, that creates bodies/contacts/etc for UK councils. A similar one could exist for ZWN, but I'll open a new ticket for that.
